scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts

```diff
@@ -654,13 +654,16 @@ export class DependencyResolverMain {
    * Returns the location where the component is installed with its peer dependencies
    * This is used in cases you want to actually run the components and make sure all the dependencies (especially peers) are resolved correctly
    */
-  getRuntimeModulePath(component: Component) {
+  getRuntimeModulePath(component: Component, inInWorkspace = false) {
```

Comment:
shouldn't this be called `isInWorkspace`?

Reply:
Thanks!!!